src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -285,6 +285,42 @@ public void DuplicateReferences ()
 			}
 		}
 
+		[Test]
+		public void DuplicateRJavaOutput ()
+		{
+			var proj = new XamarinAndroidApplicationProject {
+				PackageReferences = {
+					new Package { Id = "Xamarin.Android.Support.Annotations", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.Compat", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.Core.UI", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.Core.Utils", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.Design", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.Fragment", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.VersionedParcelable", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Android.Support.v4", Version = "28.0.0.3" },
+					new Package { Id = "Xamarin.Build.Download", Version = "0.7.1" },
```

Comment:
A comment on the devcom issue suggests that this NuGet package might be what is causing this: https://developercommunity.visualstudio.com/solutions/861372/view.html

I'll test that theory.

Reply:
Weird this also fixes it:
```diff
-new Package { Id = "Xamarin.Build.Download", Version = "0.7.1" },
+new Package { Id = "Xamarin.Build.Download", Version = "0.4.11" },
```
It appears there are more `.aar` files in use when using the newer XBD:
```
AarLibraries
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.2.0\playservicesbasement\playservicesbasement-16.2.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.0.1\playservicestasks\playservicestasks-16.0.1.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.0.1\playservicesstats\playservicesstats-16.0.1.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.1.0\playservicesbase\playservicesbase-16.1.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.0.0\playservicesphenotype\playservicesphenotype-16.0.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.3.0\playservicesmeasurementbase\playservicesmeasurementbase-16.3.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.0.0\playservicesadsidentifier\playservicesadsidentifier-16.0.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlefb-16.0.1\firebaseiidinterop\firebaseiidinterop-16.0.1.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlefb-16.1.0\firebasecommon\firebasecommon-16.1.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlefb-17.1.0\firebaseiid\firebaseiid-17.1.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlefb-16.3.0\firebaseanalyticsimpl\firebaseanalyticsimpl-16.3.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.3.0\playservicesmeasurementapi\playservicesmeasurementapi-16.3.0.aar
    C:\Users\myuser\AppData\Local\XamarinBuildDownloadCache\googlegps-16.0.0\playservicesclearcut\playservicesclearcut-16.0.0.aar
```

Reply:
Looks like we have an issue in Xamarin.Build.Download.  Tracking it here: https://github.com/xamarin/XamarinComponents/issues/771

Reply:
The difference I see between this test using XBD 0.7.1 and 0.4.11:

![image](https://user-images.githubusercontent.com/840039/71231811-049c5280-22b5-11ea-9675-34603fd90e7e.png)

Yet all the file sizes are the same:

![image](https://user-images.githubusercontent.com/840039/71231881-49c08480-22b5-11ea-81c1-cf9ac7dc62ed.png)

My theory is that XBD 0.7.1 is including duplicate resources somehow, but the java code in this example is OK.

There are extra `.flata` files:

![image](https://user-images.githubusercontent.com/840039/71231905-6c529d80-22b5-11ea-8e11-7edf99b9641b.png)

And yet somehow the output of `aapt2` (`packaged_resources` the base apk) is the same file size? I think this means that `aapt2` is merging resources and picking one set over the other.

This also has triggered the bug in Xamarin.Android I need to fix here.